lms/djangoapps/discussion/rest_api/tests/test_discussions_notifications.py

```diff
@@ -108,11 +109,14 @@ def test_html_tags_removal(self):
         <p>Script test: <script>alert("hello");</script></p>
         <p>Some other content that should remain.</p>
         """
-        expected_output = ('<p style="margin: 0">This is a link to a page.</p>'
-                           '<p style="margin: 0">Here is an image: </p>'
-                           '<p style="margin: 0">Embedded video: </p>'
-                           '<p style="margin: 0">Script test: alert("hello");</p>'
-                           '<p style="margin: 0">Some other content that should remain.</p>')
+        excepted_script_quot = 'alert(&amp;quot;hello&amp;quot;);' if django.VERSION >= (5, 0) else 'alert("hello");'
```

Comment:
In Django 5.0, `"` encoded to `&amp;quot;` to fix a security hardening, so we updated the expected_output, which contains `&amp;quot;` instead of `"`  in 5.2.